src/tools/pub/cmd/pub-add.md

```diff
@@ -39,6 +39,26 @@ $ dart pub add foo:'>2.0.0 <3.0.1'
 If `dart pub add <package>:<constraint>` is an existing dependency,
 it will update the constraint.
 
+## Adding Dependency Overrides
+
+Starting in Dart 3, `dart pub add` supports adding dependency overrides in a similar fashion to dev dependencies. To add a dependency override, use the following syntax:
+`dart pub add override:<package>:<version>`
+
+For example, to add an override for the `foo` package version `1.0.0`, run the following command:
+
+```
```

Comment:
When adding code blocks, we try to add a language or type, in this case `terminal` to allow either syntax highlighting or special styles. For example, on dart.dev, terminal code blocks are given a black background.

````suggestion
```terminal
````